rlclientlib/extensions/onnx/src/tensor_notation.h

```diff
@@ -0,0 +1,137 @@
+#pragma once
+
+#include "api_status.h"
+
+#include <algorithm>
```

Comment:
```suggestion
#include <algorithm>
#include <numeric>
```

Reply:
Because of std::accumulate (https://en.cppreference.com/w/cpp/algorithm/accumulate).
Required for build in 18.04